pt-BR/mail/chrome/messenger/preferences/viewpasswords.dtd

```diff
@@ -1,12 +1,12 @@
-<!ENTITY      windowtitle.label               "Password Manager">
-<!ENTITY      tab.signonsstored.label         "Passwords Saved">
-<!ENTITY      tab.signonsnotstored.label      "Passwords Never Saved">
+<!ENTITY      windowtitle.label               "Gerenciador de Senhas">
+<!ENTITY      tab.signonsstored.label         "Senhas salvas">
+<!ENTITY      tab.signonsnotstored.label      "Senhas nunca salvas">
 
-<!ENTITY      spiel.signonsstored.label       "Password Manager has saved login information for the following sites:">
-<!ENTITY      spiel.signonsnotstored.label    "Password Manager will never save login information for the following sites:">
+<!ENTITY      spiel.signonsstored.label       "O Gerenciador de Senhas salvou informações de login para os seguintes sites:">
+<!ENTITY      spiel.signonsnotstored.label    "O Gerenciador de Senhas nunca salvará informações de login para os seguintes sites:">
 
 <!ENTITY      treehead.site.label             "Site">
-<!ENTITY      treehead.username.label         "Username">
-<!ENTITY      treehead.password.label         "Password">
-<!ENTITY      remove.label                    "Remove">
-<!ENTITY      removeall.label                 "Remove All">
+<!ENTITY      treehead.username.label         "Nome de usuário">
+<!ENTITY      treehead.password.label         "Senha">
+<!ENTITY      remove.label                    "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY      remove.label                    "Excluir">